go/client/chat_cli_rendering.go

```diff
@@ -76,16 +102,25 @@ func (v conversationListView) show(g *libkb.GlobalContext, myUsername string, sh
 	ui := g.UI.GetTerminalUI()
 	w, _ := ui.TerminalSize()
 
+	prefixNumOfBytes, err := ConversationAliasPrefixNumOfBytes(v.aliases(), 2)
+	if err != nil {
+		return err
+	}
+
 	table := &flexibletable.Table{}
-	for i, conv := range v {
+	for _, conv := range v {
+		indexCell := flexibletable.Cell{
+			Frame:     [2]string{"[", "]"},
+			Alignment: flexibletable.Right,
+			Content: flexibletable.SingleCell{
+				Item: string(MakeFullConversationAlias(conv.Info.Id).MustShorten(prefixNumOfBytes)),
+			},
+		}
```

Comment:
this looks very similar to the code in conversationInfoListView.  Any way to reuse?  What's the difference between conversationListView and conversationInfoListView?